pyocd/probe/pydapaccess/interface/hidapi_backend.py

```diff
@@ -71,7 +71,7 @@ def get_all_connected_interfaces():
         boards = []
 
         for deviceInfo in devices:
-            product_name = deviceInfo['product_string']
+            product_name = str(deviceInfo['product_string'])
```

Comment:
Probably want to use `pyocd.utility.py3_helpers.to_str_safe()` so it works correctly on both Python 2.7 and 3.x.